dask/dataframe/groupby.py

```diff
@@ -530,6 +530,22 @@ def _finalize_std(df, count_column, sum_column, sum2_column, ddof=1):
     return np.sqrt(result)
 
 
+def _cum_agg_aligned(part, cum_last, index, columns, func, neutral):
+    align = cum_last.reindex(part.set_index(index).index, fill_value=neutral)
+    align.index = part.index
+    return func(part[columns], align)
+
+
+def _cum_agg_filled(a, b, func, neutral):
+    union = a.index.union(b.index)
+    return func(a.reindex(union, fill_value=neutral),
+                b.reindex(union, fill_value=neutral), fill_value=neutral)
```

Comment:
Am I correct in thinking that the fill value approach makes supporting max/min somewhat difficult?